src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -426,10 +426,11 @@ def test_nvmeof_config(self, _get_name, _run_cephadm, cephadm_module: CephadmOrc
 id = nvmeof.{nvmeof_daemon_id}
 
 [mtls]
-server_key = ./server.key
-client_key = ./client.key
-server_cert = ./server.crt
-client_cert = ./client.crt
+server_key = /server.key
+client_key = /client.key
+server_cert = /server.cert
+client_cert = /client.cert
+root_ca_cert = /root.ca.cert
```

Comment:
@baum @gbregman: guys, would that be ok?

We wanted to make the Root CA certificate use explicit. I think it was previously implicitly used via client_cert for the server, and server_cert for the client connections, but that works ONLY with self-signed certificates.

In other cases, it would only validate a single user (the one whose client cert is used as Root CA) from the server. Am I wrong?

Reply:
you are right. For now we decided to go that way. To be fixed later.